internal/cmd/get/get.go

```diff
@@ -43,6 +44,21 @@ func NewCommand(ctx context.CLIContext) *cobra.Command {
 	return cmd
 }
 
+func validateSemVer(versionString string) bool {
+	// Official RegEx from https://semver.org/spec/v2.0.0.html
+	const semVerRegexp = `^(?P<major>0|[1-9]\d*)\.(?P<minor>0|[1-9]\d*)\.(?P<patch>0|[1-9]\d*)` +
```

Comment:
Please change it to some out-of-the-box solution. E.g. https://github.com/Masterminds/semver